tests/accessor/accessor_default_values.h

```diff
@@ -13,6 +13,8 @@
 #include <type_traits>
 
 namespace accessor_default_values_test {
+using namespace sycl_cts;
+using namespace accessor_tests_common;
```

Comment:
It's usually better to avoid blanket `using namespace` statements, would it be possible to get rid of at least one of them and use proper namespace qualifiers where needed?

Reply:
@ProGTX  You are right, "using namespace sycl_cts" is not needed here. Removed.